self-service-1.5/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:12.9.0
+
+ENV APP_PATH app
+
+WORKDIR $APP_PATH
+
+COPY package.json yarn.lock ./
+
+RUN yarn install
+
+COPY . .
+
+CMD ["node", "."]
```

Comment:
Is this and the .dockerignore file necessary? github should be able to build the project without needing these files

Reply:
@Matt-Rohan  nvrm. I realized just now the dockerfile is directly used for github actions. will resolve & merge